development-guide/destination-connector-development-guide.md

```diff
@@ -77,8 +77,9 @@ The `DescribeTable` RPC call should report all columns in the destination table,
 - `soft` indicates that the `truncate` operation is a soft truncate operation and instead of removing the rows, the specified `delete_column` needs to be marked as `true`.
 
 ### AlterTable
-- The `AlterTable` RPC call should be used for changing primary key columns, adding columns, and changing data types.
-- However, this operation should not drop any columns even if the `AlterTable` request has a table with a different set of columns. Dropping columns could lead to unexpected customer data loss and is against Fivetran's general approach to data movement.
+The `AlterTable` RPC call should be used for changing primary key columns, adding columns, dropping columns, and changing data types.
+
+- `dropColumns`: A boolean field that indicates whether to drop columns. When `false`, do not drop any columns even if the `AlterTable` request has a different set of columns than the destination table. When `true`, drop columns that exist in the destination but are not present in the request.
```

Comment:
"do not drop any columns/drop columns" - is this an instruction for the user (or implementation) or behavior description? If the latter, I believe 3rd person should be used to describe what the call does.